<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AScanPage</class>
 <widget class="QWidget" name="AScanPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ChannelSelector" name="channelSelector" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layout2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout3">
         <item>
          <widget class="AScanInfoWidget" name="aScanInfoWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="AScanWidget" name="ascanWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>9</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BScanWidget" name="bscanWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>3</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ControlPanel" name="controlPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AScanWidget</class>
   <extends>QWidget</extends>
   <header>ui/ascanwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelSelector</class>
   <extends>QWidget</extends>
   <header>ui/channelselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>ui/controlpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BScanWidget</class>
   <extends>QWidget</extends>
   <header>ui/bscanwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AScanInfoWidget</class>
   <extends>QWidget</extends>
   <header>ui/ascaninfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
